test/addons-napi/test_error/test.js

```diff
@@ -83,56 +83,45 @@ common.expectsError(
 
 let error = test_error.createError();
 assert.ok(error instanceof Error, 'expected error to be an instance of Error');
-assert.strictEqual(error.message, 'error', 'expected message to be "error"');
+assert.strictEqual(error.message, 'error');
 
 error = test_error.createRangeError();
 assert.ok(error instanceof RangeError,
           'expected error to be an instance of RangeError');
 assert.strictEqual(error.message,
-                   'range error',
-                   'expected message to be "range error"');
+                   'range error');
```

Comment:
Nit: can you please put this (and the others) on the same line now that it doesn't exceed 80 chars?